ups/meas_base.table

```diff
@@ -3,6 +3,7 @@ setupRequired(pipe_base)
 setupRequired(daf_base)
 setupRequired(pex_config)
 setupRequired(coadd_utils)
+setupOptional(geom)
```

Comment:
I'm a bit confused as to how this can be an optional dependency if there is an unprotected import in the code above.